alarms_agent.cpp

```diff
@@ -109,16 +150,14 @@ int main (int argc, char **argv)
   init_ituAlarmTable();
 
   // Run forever
-  init_snmp("clearwater-alarms");
+  init_snmp_handler_threads("clearwater-alarms");
```

Comment:
We used to call `netsnmp_ds_set_boolean`, `init_agent` and `init_snmp`. Now we call `snmp_setup` and `init_snmp_handler_threads`. I know next to nothing about netsnmp but that seems like quite a significant change. Is it right?